data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -238,9 +242,31 @@ def _create_tile_data(
     def _create_downloadable_data(
         self, score_county_state_merged_df: pd.DataFrame
     ) -> pd.DataFrame:
-        return score_county_state_merged_df[
+        df = score_county_state_merged_df[
             constants.DOWNLOADABLE_SCORE_COLUMNS
-        ]
+        ].copy(deep=True)
+
+        df_of_float_columns = df.select_dtypes(include=["float64"])
+
+        for column in df_of_float_columns.columns:
+            # TODO: create a schema for fields to make it more explicit and safe which
+            #  fields are percentages.
+            if any(x in column for x in constants.PERCENT_PREFIXES_SUFFIXES):
+                # Convert percentages from fractions between 0 and 1 to an integer
+                # from 0 to 100.
+                df_100 = df[column] * 100
+                df_int = np.floor(
+                    pd.to_numeric(df_100, errors="coerce")
+                ).astype("Int64")
+                df[column] = df_int
```

Comment:
@esfoobar-usds 

This is another way that simplifies the above code. See below for the output.

` df.loc[:, column] = (100 * df.loc[:, column]).round(0).astype("Int64")`

![Screen Shot 2022-01-13 at 8 35 00 AM](https://user-images.githubusercontent.com/94847739/149339888-3f5b7cc8-07a6-4fd7-b7fb-6234d364d83a.png)